app/shared/app-data/src/commonMain/kotlin/data/models/preference/UISettings.kt

```diff
@@ -30,6 +30,7 @@ data class UISettings(
     val myCollections: MyCollectionsSettings = MyCollectionsSettings.Default,
     val searchSettings: SearchSettings = SearchSettings.Default,
     val episodeProgress: EpisodeProgressSettings = EpisodeProgressSettings.Default,
+    val startupToWelcomeWizard: Boolean = true,
```

Comment:
```suggestion
    val onboardingCompleted: Boolean = true,
```
"wizard" 意为向导, 不一定指代新手引导功能. 所以可以把特定于新手引导的改成叫 onboarding.